pkg/steps/artifacts.go

```diff
@@ -413,6 +415,51 @@ func addPodUtils(pod *coreapi.Pod, artifactDir string, decorationConfig *prowv1.
 
 	pod.Spec.Volumes = append(pod.Spec.Volumes, logVolume, toolsVolume)
 	pod.Spec.Volumes = append(pod.Spec.Volumes, blobStorageVolumes...)
+
+	if clone {
+		codeMount, codeVolume := decorate.CodeMountAndVolume()
```

Comment:
Is there going to be anything messy with having code and $HOME on different devices that we wouldn't already have hit with the current setup?

Reply:
I don't think so